### ISSUE TITLE
Keyboard access to popover

### ISSUE THREAD

**Issue by vanch3d** [SAFeSEA/openEssayist-slim#3]
Also on Your Draft : Key Words & Key Phrases the key words receive keyboard focus, but they cannot be operated and so the pop-up info cannot be accessed
